libs/sdk-core/src/greenlight/node_api.rs

```diff
@@ -1435,6 +1435,67 @@ impl NodeAPI for Greenlight {
         debug!("send_custom_message returned status {:?}", resp.status);
         Ok(())
     }
+
+    // Gets the routing hints related to all private channels that the node has
+    async fn get_routing_hints(&self) -> NodeResult<Vec<RouteHint>> {
+        let mut hints: Vec<RouteHint> = vec![];
+        let mut node_client = self.get_node_client().await?;
+        let peers = node_client
+            .list_peers(cln::ListpeersRequest::default())
```

Comment:
This should be `list_peer_channels`. `list_peers` might also returns channels, but that functionality is deprecated and will be removed soon. (is already removed if deprecated apis are not enabled)